test/online_test.py

```diff
@@ -616,6 +616,16 @@ def test_export(self):
         self.worksheet.clear()
         self.spreadsheet.del_worksheet(worksheet_2)
 
+    def test_sort_range(self):
+        self.update_values("A1",2)
```

Comment:
Use update values and get_values to get and set values in one-go. This will reduce the test run time. Also add tests for descending and sort_index.